src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+@import "variables.scss";
+
+.card {
+  margin: 0;
+  padding: 32px 16px 0;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    margin: 0 3px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $second-text-color;
+  }
+
+  &__code--position {
+    margin-top: 4px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__rate--position {
+    margin-top: 16px;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price--position {
+    margin-top: 24px;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    background-color: $button-color;
+    border: none;
+    border-radius: 5px;
+  }
+
+  &__buy:hover {
+    color: $button-color;
+    background-color: #fff;
+    border: 1px solid $button-color;
+  }
+
+  &__buy--position {
+    margin: 16px 0;
+  }
+}
+
+.price {
+  color: $second-text-color;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.sum {
```

Comment:
Would be better to use correct classes according to BEM like `card__price-sum` and use element nesting in scss